config/locales/find.yml

```diff
@@ -403,7 +403,7 @@ en:
             other: "%{formatted_count} courses found"
   helpers:
     legend:
-      courses_query_filters:
+      courses_search_form:
```

Comment:
I renamed because I realised courses_query_filters was not a good name. So changed the same name of the form object.